### ISSUE TITLE
TypeError: can't pickle module objects

### ISSUE THREAD

**Issue by YumingShao** [wepe/MachineLearning#16]
File "/home/shaoym/anaconda2/lib/python2.7/copy_reg.py", line 70, in _reduce_ex
    raise TypeError, "can't pickle %s objects" % base.__name__

TypeError: can't pickle module objects

楼主，我在执行您的cnn.py代码时，这个pickle module报上面的错误，不知是什么原因？
不胜感激！

**Comment by Tracy-yin** [comment]
我也遇到了 怎么解决呢

**Comment by hangzhang2016** [comment]
嗯，好问题，求同！！

**Comment by hangzhang2016** [comment]
yes, this is a good question which I also meet recently, good luck.
